app/javascript/controllers/new_message_controller.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
-
```

Comment:
Based on my reading, this isn't needed in ES6 modules. Apparently they use strict by default